llvm/test/CodeGen/RISCV/pr64935.ll

```diff
@@ -4,10 +4,7 @@
 define i1 @f() {
 ; CHECK-LABEL: f:
 ; CHECK:       # %bb.0:
-; CHECK-NEXT:    lui a0, 524288
-; CHECK-NEXT:    not a0, a0
-; CHECK-NEXT:    sltiu a0, a0, 2
-; CHECK-NEXT:    xori a0, a0, 1
+; CHECK-NEXT:    li a0, 1
```

Comment:
For the avoidance of doubt, this is semantically equivalent:
https://alive2.llvm.org/ce/z/8gY545